lib/ferto/client.rb

```diff
@@ -54,13 +54,23 @@ def initialize(opts = {})
     #   resource
     # @param aggr_id       [String] aggregation identifier
     # @param aggr_limit    [Integer] aggregation concurrency limit
+    # @param aggr_proxy    [String] the HTTP proxy to use for downloading the
+    #   resource, by default no proxy is used
```

Comment:
Make an explicit note that this is stored on the aggregation layer and cannot be changed after creation ( at least immediately ).